### PR TITLE
Bug 1199671 - Fixed topLayoutGuide constraint on BrowserViewController for iOS9

### PR DESCRIPTION
**Pull request by sleroux**

### CODE REVIEW

File: Client/Frontend/Browser/BrowserScrollController.swift [hunk 1]
```diff
@@ -37,12 +37,14 @@ class BrowserScrollingController: NSObject {
     weak var snackBars: UIView?
 
     var footerBottomConstraint: Constraint?
-    var headerTopConstraint: Constraint?
+    // TODO: Since SnapKit hasn't added support yet (Swift 2.0/iOS 9) for handling layoutGuides,
+    // this constraint uses the system abstraction instead of SnapKit's Constraint class
+    var headerTopConstraint: NSLayoutConstraint?
```
Comment (thebnich) [inline Client/Frontend/Browser/BrowserScrollController.swift]: Found [this](https://github.com/SnapKit/SnapKit/commit/6252444966517bd22c9bb673c8a41d8d1c7306ea). Does that mean we could use `snp_topLayoutGuideTop`?

Comment (sleroux) [inline Client/Frontend/Browser/BrowserScrollController.swift]: Oh snap - I didnt know that existed! Yes, we can use that but it looks like it's broken on SnapKit's swift-2.0 branch but there is a PR open for this here: https://github.com/SnapKit/SnapKit/pull/137. I'm going to file a follow up bug to change these back to Snapkit after that patch is merged in.

**Comment by thebnich** [comment]
OK, cool :+1:

File: Client/Frontend/Browser/BrowserScrollController.swift (hunk 1 above)
Comment (thebnich) [inline Client/Frontend/Browser/BrowserScrollController.swift]: > Oh snap -

punny

**Comment by sleroux** [comment]
I've opened https://bugzilla.mozilla.org/show_bug.cgi?id=1200210 to track updating this back to Snapkit.

**Pull request merged**